.travis.yml

```diff
@@ -1,30 +1,15 @@
-sudo: false
 language: python
 matrix:
   include:
-  - python: 2.6
-    env: TOXENV=py26
-  - python: 2.7
-    env: TOXENV=py27
-  - python: 3.3
-    env: TOXENV=py33
   - python: 3.4
     env: TOXENV=py34
   - python: 3.5
     env: TOXENV=py35
   - python: 3.6
     env: TOXENV=py36
-  - python: pypy
-    env: TOXENV=pypy
-  allow_failures:
-  - python: 2.6
-    env: TOXENV=py26
-  - python: 2.7
-    env: TOXENV=py27
-  - python: 3.3
-    env: TOXENV=py33
-  - python: pypy
-    env: TOXENV=pypy
+  - python: 3.7
+    env: TOXENV=py37
+    dist: xenial
```

Comment:
Do we need the distribution specified?

Reply:
Yes, Python 3.7 requires Xenial on Travis CI.